packages/sync-service/lib/electric/postgres/inspector/ets_inspector.ex

```
@@ -364,6 +364,8 @@ defmodule Electric.Postgres.Inspector.EtsInspector do
       :not_in_cache -> :not_in_cache
       oid -> {:ok, {oid, relation}}
     end
+  rescue
+    ArgumentError -> :not_in_cache
```

If the ETS table is not existing then then the process is not going to exist, so won't this mean that the next call errors with noproc? Would it not be better to return an error and not try to call the process?